lib/rubocop/cop/bugcrowd/replica_identity_required.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Bugcrowd
+      class ReplicaIdentityRequired < RuboCop::Cop::Cop
+        # Checks that tables have a replica identity defined
+        #
+        # @example
+        #
+        # # bad
+        #  ```
+        #  def change
+        #    create_table :new_table_name, id: :uuid do |t|
+        #      t.text :name, null: false
+        #    end
+        #  end
+        #  ```
+        #
+        # # good
+        #  ```
+        #  def change
+        #    create_table :new_table_name, id: :uuid do |t|
+        #      t.text :name, null: false
+        #    end
+        #    set_replica_identity(:full)
+        #  end
+        #  ```
+
+        MSG = 'tables should have `set_replica_identity` called after `create_table`, ' \
+         'probably with :identity = :full'
```

Comment:
```suggestion
         'probably with set_replica_identity(:full)'
```

just so people can copy pasta instead of thinking too hard

Reply:
Done, also went a step further and added an autocorrector.